scene/gui/texture_progress_bar.cpp

```diff
@@ -608,11 +608,10 @@ int TextureProgressBar::get_fill_mode() {
 }
 
 void TextureProgressBar::set_radial_initial_angle(float p_angle) {
-	while (p_angle > 360) {
-		p_angle -= 360;
-	}
-	while (p_angle < 0) {
-		p_angle += 360;
+	ERR_FAIL_COND_MSG(!Math::is_finite(p_angle), "Angle is non-finite.");
+
+	if (p_angle < 0.0 || p_angle > 360.0) {
+		p_angle = Math::fposmodp(p_angle, 360);
```

Comment:
```suggestion
		p_angle = Math::fposmodp(p_angle, 360.0);
```
For consistency with above line.

Reply:
With just this it will lead to compilation error.
```
 scene\gui\texture_progress_bar.cpp(614): error C2666: 'Math::fposmodp': overloaded functions have similar conversions
.\core/math/math_funcs.h(251): note: could be 'double Math::fposmodp(double,double)'
.\core/math/math_funcs.h(243): note: or       'float Math::fposmodp(float,float)'
scene\gui\texture_progress_bar.cpp(614): note: while trying to match the argument list '(float, double)'
```
So I'll add explicit casting.

Reply:
Then it should be just
```suggestion
		p_angle = Math::fposmodp(p_angle, 360.0f);
```